### DISCUSSION TITLE
Replacing initcontainer with custom dind image container

### DISCUSSION THREAD

**Discussion by rekha-prakash-maersk** [actions/actions-runner-controller#3464] [Questions]
The pods with initcontainer takes 1 minute to start up, whereas the pod with only 2 container ( runner and **custom dind image**) only takes 10 seconds to spin up. I like to understand is there any specific reason behind having this initcontainer ?

**Comment by JohnYoungers** [comment]
I believe the init container is just copying files from the runner container to the dind container.  See this thread: https://github.com/actions/actions-runner-controller/discussions/2944
